tests/unit/core/test_tracking.js

```diff
@@ -48,6 +48,18 @@ describe('Tracking', () => {
     ).toBe(true);
   });
 
+  it('should log OFF when not enabled due to Do Not Track', () => {
+    tracking = new Tracking({
+      _isDoNotTrackEnabled: () => true,
+      _log: {
+        info: sinon.stub(),
+      },
+      trackingEnabled: true,
```

Comment:
For example, it would be pretty hard to add coverage for the config logic here. Again, not important enough to block on.